src/express/commands/monitor.js

```diff
@@ -58,6 +58,12 @@ async function checkCheckpoint(ip) {
   return 0
 }
 
+export async function checkLatestMilestone(ip) {
+  const url = `http://${ip}:1317/milestone/latest`
+  const response = await fetch(url)
+  return await response.json()
```

Comment:
Not sure here tbh, but are all the `await` necessary?
I see we are awaiting on `fetch(url)`, `response.json()` and when the function itself is called `milestone = await checkLatestMilestone(borHosts[0])`

Reply:
I think they're required. For fetch, it's a network call so we'll have to wait. `.json()` function returns a promise and does not return the required result immediately hence we'd need an await there. Moreover, due to this the function `checkLatestMilestone` returns a promise so we'll have to add an await before calling it. Wdyt?

Reply:
Yeah, right, makes sense, thanks